minimini/app/controllers/tasks_controller.rb

```diff
@@ -1,12 +1,10 @@
 class TasksController < ApplicationController
-  protect_from_forgery
   before_action :set_task, only: [:show, :edit, :update, :destroy]
+  before_action :require_login
 
   def new
     @task = Task.new()
```

Comment:
引数がない場合は、カッコはいらないですね
```suggestion
    @task = Task.new
```

Reply:
()は書く必要ないんですね。対応しました。